vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -42,19 +42,18 @@ BatPublishers::~BatPublishers() {
 }
 
 void BatPublishers::calcScoreConsts(const uint64_t& min_duration) {
-  uint64_t min_duration_ms = min_duration *
-      braveledger_ledger::_milliseconds_second;
-  a_ = (1.0 / (braveledger_ledger::_d * 2.0)) - min_duration_ms;
+  uint64_t min_duration_big = min_duration * 100;
```

Comment:
we needed to reduce this from 1000 to 100 as we were hitting limit and getting `nan`

Reply:
I'm confused, are you talking about the JS side? Presumably multiplying by 1000 was for ms, so why would we multiply by 100?

Reply:
correct 1000 was for ms. If we remove multiply completely formula is not that efficient anymore, so that's why we still multiply it with 100 to have better results